Xamarin.Forms.Core/NativeBindingHelpers.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Xamarin.Forms.Internals;
+
+using static System.String;
+
+namespace Xamarin.Forms
+{
+	static class NativeBindingHelpers
```

Comment:
This is the meaty part to review. It's actually the only important part to review as well.